src/qmoperators/two_electron/ExchangePotentialD1.h

```diff
@@ -37,6 +37,7 @@ class ExchangePotentialD1 final : public ExchangePotential {
     Orbital calcExchange(Orbital phi_p);
     void calcInternal(int i);
     void calcInternal(int i, int j);
+    void calcInternal(int i, int j, Orbital &phi_i, Orbital &phi_j);
```

Comment:
Why do you need an extra calc_internal routine? The orbitals can be retrived with Phi[i] and Phi[j]?

Reply:
I just copied this from the master version. This will change completely anyway when the "new Poisson integrals" will be included.